lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/v2x/etsi/DenmContent.java

```diff
@@ -79,13 +79,12 @@
      */
     private final String extendedContainer;
 
-    public DenmContent(final long time, final GeoPoint senderPosition, final String eventRoadId, final SensorType warningType, final int eventStrength, final float causedSpeed, final float senderDeceleration) {
+    public DenmContent(final long time, GeoPoint senderPosition, final String eventRoadId, final SensorType warningType, final int eventStrength, final float causedSpeed, final float senderDeceleration) {
```

Comment:
No need to remove the final keyword here. `final` just tells, that the paramater `senderPosition` passed to the method cannot be changed (re-assigned) within the body of the method/constructor.